### ISSUE TITLE
Only one typesetting comment ($t) is allowed per database

### ISSUE THREAD

**Issue by MostAwesomeDude** [metamath/metamath-exe#161]
Quoting [the PDF](https://us.metamath.org/downloads/metamath.pdf), p144:

> In version 0.177 of the Metamath program, there may be only one `$t` comment in a database. This restriction may be lifted in the future to allow many `$t` comments in a database.

I would like to lift this restriction.

I am generating part of a database from structured data. I've arranged my generated data so that it can be appended to the end of an existing database. My generated data includes labels which are not in the main database, and I'd like to generate a typesetting comment (`$t`) so that those labels can be correctly formatted.

I'm planning on working around this for now, since my build step allows for aggressive `sed` incantations, but I'd prefer if I were able to merely have multiple `$t` which are monoidally aggregated.

**Comment by digama0** [comment]
Agreed, this is a silly restriction which is not present for `$j` comments and necessitates some very inconvenient separation of typesetting info from constant declarations.

**Comment by tirix** [comment]
Agreed - the code to change appears to be in [`mmwtex.c`](https://github.com/metamath/metamath-exe/blob/a7168a0387373a01e213f18f4090b2290efc3467/src/mmwtex.c#L315), in the `readTexDefs` function.
